src/test/java/com/google/graphgeckos/dashboard/GitHubControllerTests.java

```diff
@@ -0,0 +1,55 @@
+package com.google.graphgeckos.dashboard;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import com.google.graphgeckos.dashboard.fetchers.github.GitHubController;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(controllers = GitHubController.class)
+@AutoConfigureMockMvc
+public class GitHubControllerTests {
```

Comment:
FYI: The tests you do in this class are called golden file tests (because the results are compared to a result file which is often called "golden file").

Reply:
Thanks for the information!